app/help.py

```diff
@@ -96,5 +96,6 @@
   See readme.md for an introduction
   Both files may be found in "%s"
   Please send feedback and bug reports to dschuyler@
+  (Release build)
```

Comment:
Oh, this is an intentional difference between the master and release branches. It's purpose is to tell us when a user is using a stable release build vs a non-stable master branch build. Please don't add this to the master branch.